.gitlab/ci/on-push.yml

```
@@ -47,6 +47,31 @@ run-validations:
   rules:
     - if: '$NIGHTLY'
 
+# runs in gitlab for the on-push flow, on every new commit pushed to the branch.
+validate-content-conf:
+  stage: unittests-and-validations
+  needs: []
+  extends:
+    - .default-job-settings
+  rules:
+    - if: '$CI_PIPELINE_SOURCE =~ /^(push|contrib)$/ && $CI_COMMIT_BRANCH !~ /^[0-9]{2}\.[0-9]{1,2}\.[0-9]$/'
```

```suggestion
    - if: '$CI_PIPELINE_SOURCE =~ /^(push)$/ && $CI_COMMIT_BRANCH !~ /^[0-9]{2}\.[0-9]{1,2}\.[0-9]$/'
```
this step is not relevant for contributions.